web/packages/teleport/src/Discover/SelectResource/SelectResource.story.tsx

```diff
@@ -55,20 +67,45 @@ export const PartialAccess = () => {
   );
 };
 
+export const PreferredDBAccess = () => {
+  return (
+    <Provider resources={[3]}>
+      <SelectResource onSelect={() => null} />
+    </Provider>
+  );
+};
```

Comment:
use the enum key instead of its num? i was looking at this story and didn't think it is working 🤔 , i think it's a story issue b/c it seems to work fine on the actual app
<img width="1588" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/e20fabd0-2332-4e74-93b3-c7fb74250673">